index.bs

```diff
@@ -407,6 +412,15 @@ This section defines the MIAF AV1 Baseline profile of [[!HEIF]], specifically fo
 
 If the brand <code>MA1B</code> is in the list of [=compatible_brands=] of the [=FileTypeBox=], the common constraints in the section [[#brands]] shall apply.
 
+The following shared conditions and requirements from [[!MIAF]] shall apply:
```

Comment:
This is how it's done in MIAF. You can create a generic MIAF file that does not adopt these constraints, but if you want to use the specific profiles you're promising that you're adhering to these restrictions.

This means we can create a generic AVIF file that does not need to adhere to these restrictions. But if you want to use a specific profile you're promising that the file is well-behaved.